Jellyfin.Plugin.NextPVR/Jellyfin.Plugin.NextPVR.csproj

```diff
@@ -17,7 +18,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.*-*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.9.6" />
```

Comment:
Please revert

Reply:
OK I have to watch for this, VS did this automatically.